model/src/main/java/org/mskcc/cmo/metadb/model/MetadbSample.java

```diff
@@ -94,10 +95,6 @@ public void addSampleMetadata(SampleMetadata sampleMetadata) {
         sampleMetadataList.add(sampleMetadata);
     }
 
-    public void setPatientUuid(UUID uuid) {
```

Comment:
didn't need it here since `MetadbPatient` itself is already a member of `MetadbSample`